libraries/service_helpers.rb

```diff
@@ -0,0 +1,58 @@
+module CernerSplunk
+  # Mixin Helper methods for the Splunk Service resource
+  module ServiceHelpers
+    include ResourceHelpers
+    require 'win32/service' if RUBY_PLATFORM =~ /mswin|mingw|windows/
+
+    # Reports if the Windows service for the current package exists
+    # @return [Boolean]
+    def service_exists
```

Comment:
I had an issue stubbing this method with a question mark, but I'll give it another shot